vocabularies/voc4cat/0007025.ttl

```diff
@@ -0,0 +1,14 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<https://w3id.org/nfdi4cat/voc4cat_0007025>
+    a skos:Concept ;
+    dcterms:identifier "voc4cat_0007025"^^xsd:token ;
+    skos:definition "A reaction rate that is normalized not to the total mass of catalyst, but either to mass of active ingredient, surface area or certain measured adsorption sites."@en ;
```

Comment:
Please exclude the "not ... but" part. It is difficult / not possible to define something by saying what it is not.